### DISCUSSION TITLE
Only write specific tag (genre) to ID3 tags.

### DISCUSSION THREAD

**Discussion by porterdcarnegie** [beetbox/beets#4346] [General]
I have a music library of which I manually have written all the ID3 tags.
I wish beets to only write genre, to not overwrite any other tags with what it thinks fits better.
Can that be accomplished?

**Comment by arsaboo** [comment]
There is an [open issue](https://github.com/beetbox/beets/issues/4087) related to this feature.

**Reply by porterdcarnegie** [reply]
Thank you, managed to get only genre tag written with the help of the draft by jOj0.
